VSRAD.Package/VSRAD.Package.csproj

```diff
@@ -242,6 +242,7 @@
     <Compile Include="ToolWindows\EvaluateSelectedWindow.cs" />
     <Compile Include="ToolWindows\SliceVisualizerWindow.cs" />
     <Compile Include="Utils\ArrayRange.cs" />
+    <Compile Include="Utils\RecentlyUsedCollection.cs" />
     <Compile Include="Utils\CollectionExtensions.cs" />
     <Compile Include="Utils\MruCollection.cs" />
```

Comment:
As far as I can tell, `MruCollection` behaves the same, sans item pinning. It may be reasonable to merge `RecentlyUsedCollection` and `MruCollection` since the latter is only used for the target hosts dropdown, and we may want to add item pinning to it in the future as well.

Reply:
It seems reasonable, however I'm not sure that this is possible. The `InsertItem` override is rely on fact that we store `LastUsed` items in collection, and this is the only way to properly initialize the `_pinnedCount` variable, when we parsing the config Json file. So I can't really have the collection with generic item type like `MruCollection` in this case

Reply:
https://github.com/vsrad/radeon-asm-tools/pull/187#discussion_r594210421 - thanks, fixed naming in 1f77160